build.cake

```diff
@@ -1,4 +1,8 @@
 #tool nuget:?package=NUnit.ConsoleRunner&version=3.6.1
```

Comment:
We are back to using both tabs and spaces for indention in this file.

Reply:
Yes. I should try to tell VS how to handle this file.